app/views/projects/_project.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="col-md-6 project-item">
```

Comment:
This partial isn't necessary, is it? We move all this partial to react component